new/BLorient13156.xml

```diff
@@ -0,0 +1,154 @@
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 13156</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">161</measure>
+                                    <measure unit="page" type="blank">46r, 112r, 127r, 144r</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>115</height>
+                                        <width>92</width>
+                                        <depth/>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="12 15"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <date notBefore="1400" notAfter="1500" resp="PRS8999Strelcyn"/>
+                                <desc/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote type="miniature" xml:id="d1">
+                                <locus target="#4v"/>
+                                <desc>The Virgin and Child</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d2">
+                                <locus target="#46v"/>
+                                <desc>Michael the Archangel</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d3">
+                                <locus target="#62v"/>
+                                <desc>[Our Lady] at prayer while standing</desc>
```

Comment:
Could you possibly use mark up instead of []? Are the contents of desc in these descriptions translations of captions (if so, it could be indicated) or your descriptions?

Reply:
They are adaptions of Strelcyn's translations of the captions. I didn't want to type in the complete captions at the moment

Reply:
Thanks @karljonaskarlsson. There is also always the option of assigning ATs to these when possible, but this is a choice of each encoder, so entirely up to you. Thanks for the work.